srcpkgs/corosync/INSTALL.msg

```diff
@@ -0,0 +1,6 @@
+An example configuration file was placed into /etc/corosync/ for test purpose,
```

Comment:
This file should be dropped.  We are not npm, so we don't have chatty install messages.  The use of INSTALL.MSG files is reserved for when action must be taken to not break the system, or when additional packages must be installed that require user review.